utils/helpers.ts

```diff
@@ -1,4 +1,10 @@
 import { compile } from 'handlebars';
 import { Context } from '../common/notification/types';
+import { getLogger } from '../common/logger/logger';
 
-export const renderTemplate = (template: string, context: Partial<Context>, strict: boolean = false) => compile(template, { strict })(context);
+export const renderTemplate = (template: string, context: Partial<Context>, strict: boolean = false) => {
+    if (!template) {
+        return undefined;
```

Comment:
Can the template be empty by exception or is the system designed like that?
If it's an exception I would propose to add a log message to keep track of it.